master/security/high-connection-workloads.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Enable extreme high-connection workloads 
+---
+
+### Big Picture
+
+Use a Calico network policy rule to bypass Linux conntrack for traffic to extreme high-connection workloads.
+
+### Value
+
+When extreme high-connection workloads exceed the number of connections that Linux conntrack can track, connections can be rejected or dropped. Calico network policy can be used to selectively bypass Linux conntrack for traffic to/from these types of workloads.
```

Comment:
I think this should say "When the number of connections on a node exceeds..."

Reply:
Fixed